event-stream-processing/package.json

```diff
@@ -24,6 +24,7 @@
     "rm-coverage": "rimraf coverage",
     "pack": "npm-run-all mkdirs rm-pack compile build-pack",
     "test": "jest --coverage",
+    "jest": "jest",
```

Comment:
The jest coverage option should have been moved into the config file. So, this should actually do the same thing as 'test'.

Reply:
@TimCsaky, on a second thought, @mbystedt is right. you can still run a single test + code coverage with:
```
$ npm run test -- --watch src/node-css-parser.test.ts
```
so, please remove this line.